cfr.yaml

```diff
@@ -32,6 +32,34 @@
     id: contentBlocking
     params:
       - ContentBlockingMilestone
+- id: MILESTONE_MESSAGE_2
```

Comment:
This should instead reference the Firefox version it targets, that's what our current convention looks like.

Reply:
Oh yep, I see